ChemoTracker/styles/profile-main.js

```diff
@@ -169,8 +172,7 @@ const styles = StyleSheet.create({
   profileButton: {
     color: "#FFFFFF",
     backgroundColor: 'transparent',
-    fontSize: 17,
-    marginRight: 15
+    fontSize: 20,
```

Comment:
It looks very big on android, so i changed it to 17 and i tested it with Seun's phone
also the marginRight is needed for the save button cause it hits the edge on android